otelconfig/otelconfig_test.go

```diff
@@ -72,6 +72,9 @@ func (logger *testLogger) requireNotContains(t *testing.T, expected string) {
 	}
 }
 
+var trueVal = true
+var falseVal = false
```

Comment:
Note/question for reviewer: this feels weird. Is this weird? I wasn't sure how else would be better to specify the opposite of the true pointer.

Reply:
This is what you have to do with pointers. It's one reason why the pointer model is annoying, but it works.

Reply:
Ah okay thanks for confirming!